lib/services/local/trust.go

```diff
@@ -213,9 +213,6 @@ func (s *CA) GetCertAuthority(id services.CertAuthID, loadSigningKeys bool, opts
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	if err := ca.Check(); err != nil {
-		return nil, trace.Wrap(err)
-	}
```

Comment:
Why just remove the checks here?

Same question elsewhere this is done.

Reply:
@russjones IIUC, We don't need to validate values already in the database, so I think this call is unnecessary. Additionally, Unmarshal functions already call the `CheckAndSetDefaults` methods making it partially redundant.

Similar reasoning for using `CheckAndSetDefaults` instead of `Validate` in other spots.